pkg/descriptions.json

```diff
@@ -1975,6 +1975,13 @@
       "System.Text.Json.Utf8JsonReader"
     ]
   },
+  {
+    "Name": "Microsoft.Bcl.Json.Sources",
+    "Description": "Provides the source code implementation of System.Text.Json as internal types. This package is only required for projects targeting .NET Framework 4.6.1+ or .NET Core 2.x; and is not for projects that can target .NET Core 3.0 and higher.",
+    "CommonTypes": [
+      "System.Text.Json.Utf8JsonReader"
```

Comment:
While you are updating this, can you please add "System.Text.Json.Utf8JsonWriter" as well to the list of `CommonTypes`?

Also here (I forgot to add it):
https://github.com/dotnet/corefx/blob/54f3a59c5e0cd0db90e9c74cd11eebfd28f028b7/pkg/descriptions.json#L1975

Reply:
sure.